content/articles/introduction-to - application-performance-management/index.md

```diff
@@ -0,0 +1,82 @@
+
+### Measuring performance using APM
+APM is used to rate performance based on two metrics; user performance experience and computational resources utilized by an application. An example of user performance metrics is an application's response time at peak load. Load is defined as the volume of digital transactions that a computer application performs. These transactions are initiated through computer-based demands.
+
+During the development of computer programs, the applications may not be subjected to peak loads and hardly experience performance limitations. However, this may not be the case with the end-user. At peak loads, applications may take longer to process demands. The period between a computer-based demand and the application's feedback to the user is referred to as response time. Inefficient and poorly managed applications will result in a longer response time.
+
+An example of a computational resource metric used to measure performance is storage. Computer programs use the device's hardware and software to get tasks done. As so is the case, the computer should possess the capacity to hold and run an application's load at its peak without being compromised. APM applies this metric to identify and provide solutions to computer resources-related application problems.
+
+### Dimensions of APM
+[Gartner](https://www.gartner.com/en), a global research and advisory firm for Information Technology (IT), finance, and Human Resource (HR), states that application performance management is based on three major dynamics. These include:
+
+-   Digital Experience Monitoring (DEM)
+
+-   Application Discovery, Tracing, and Diagnostics (ADTD)
+
+-   Artificial Intelligence for IT Operations (AIOps)
+
+Digital Experience Monitoring (DEM) focuses on the relationship between users and applications. The data it collects and analyses are based on how users get affected by the application in use. It detects and helps solve problems that affect user experience directly. Examples of such problems include long application response time and user unfriendliness.
+
+On the other hand, Application Discovery, Tracing, and Diagnostics (ADTD) address the technical logistics of an application. This includes the runtime architecture. It manages applications by monitoring both hardware and software resources that form the digital environment to execute an application.
+
+Artificial Intelligence for IT Operations (AIOps) applies to platforms that analyze and learn machine language to enhance the performance of applications. It improves performance continuously by integrating and deploying core IT functions to suit different and more sophisticated tasks.
```

Comment:
Interesting stuff here. Consider delving a bit deeper into these dimensions of APM.

Reply:
this is well noted @collins-a . i will work on it and get back to you shortly.
Thanks for the positive feedback.